YoCode/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "YoCode": {
       "commandName": "Project",
-      "commandLineArgs": "C:\\Users\\ukmzil\\source\\repos\\Tests-Sent-by-People\\Real\\drew-gibbon C:\\Users\\ukmzil\\source\\repos\\junior-test"
+      "commandLineArgs": "C:\\Users\\ukmaug\\Downloads\\yes-to-interview\\4C185EF91A6D3588483A9B5957C66F125445D68A6455BAC16E9290F87991E7F9 C:\\Users\\ukmaug\\Downloads\\junior-test"
```

Comment:
Fighting over this path is getting a little old. Perhaps `launchSettings.json` should be removed from git and added to `.gitignore` so that everyone can set their own parameters? Otherwise it's going to end up with like 8000 commits over the course of the project.